src/connectors/normalize.js

```diff
@@ -0,0 +1,20 @@
+const vec3 = require('../math/vec3')
+
+const fromPointAxisNormal = require('./fromPointAxisNormal')
+
+/**
+ * Normalize the given connector, calculating new axis and normal
+ * @param {Connector} connector - the connector to normalize
+ * @returns {Connector} a new connector
+ */
+const normalize = (connector) => {
```

Comment:
I don't understand the need for this function as connectors are created by only on function; fromPointAxisNormal().
And that function converts the given Axis and Normal to unit vectors.
Any reason to keep this?

Reply:
I think it was/is needed in other use cases (been a while since I looked at details): can we please keep for now as well & add to cleanup pass ?

Reply:
Ok. But I won’t add tests until we find a use case.